modules/abstract-cosmos/src/lib/utils.ts

```diff
@@ -671,6 +705,35 @@ export class CosmosUtils implements BaseUtils {
   isValidAddress(address: string): boolean {
     throw new NotImplementedError('isValidAddress not implemented');
   }
+
+  /**
+   * Validates if the address matches with regex @see contractAddressRegex
+   * @param {string} address
+   * @returns {boolean} - the validation result
+   */
+  isValidContractAddress(address: string): boolean {
+    throw new NotImplementedError('isValidContractAddress not implemented');
```

Comment:
let's implement using the length comparison and add ToDo to implement with Regex. Else it will be blocker in the platform side.

Reply:
it is there in extended classes